share/spice/zomato/zomato.js

```diff
@@ -0,0 +1,149 @@
+(function (env) {
+    "use strict";
+    var query_orig = DDG.get_query();
+    
+    //$.getJSON('http://dheerajavvari.zdev.net/duckducktrack.php?value='+encodeURIComponent(query_orig));
+    
+    env.ddg_spice_zomato = function(api_result){
+        
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error) {
+            return Spice.failed('zomato');
+        }
+        
+        var res, loc_link_str, address_link_str, data, hours,
+            estd = '';
+
+        // Render the response
+        // console.log(api_result);
+        //DDG.require('maps', function(){
+            //DDG.require('moment.js', function(){
+            //console.log(api_result);
+                Spice.add({
+                    id: "zomato",
+                    name: "Places",
+                    model: "Place",
+                    view: "Places",
+                    signal: "high",
+                    data: api_result.restaurants,
+                    allowMultipleCalls: (api_result.next)?true:false,
+                    meta: {
+                        sourceName: "Zomato",
+                        sourceUrl: api_result.more_at,
+                        snippetChars: 110,
+                        sourceIcon: true,
+                        sourceIconurl: 'https://www.zomato.com/images/logo/zomato_favicon3.png',
+                        itemType: 'Places',
+                        searchTerm: query_orig,
+                        next: api_result.next
```

Comment:
I think you'll want to fallback to `false` when there isn't a `next` defined? `api_result.next || false`